packages/block-library/src/navigation/edit/index.js

```diff
@@ -626,10 +626,11 @@ function Navigation( {
 								</div>
 							</>
 						) }
-						<h3>{ __( 'Overlay Menu' ) }</h3>
```

Comment:
I'd be curious (and also a bit scared) to see how many more `h3`s are being used as labels (since the [styles](https://github.com/WordPress/gutenberg/blob/0f1271245f4e634e65781219250c0c1a5c9d60ab/packages/interface/src/components/complementary-area/style.scss#L47-L54) are the same)

Reply:
😅

May be alleviated if we add fieldset/legend support to BaseControl (or provide a new component for it)? Like the `h3` below this is used as a heading for multiple ToggleControls.

Reply:
Yeah, it definitely sounds like we need to gather and brainstorm all requirements that we'd like for a new version of `BaseControl`. Support for fieldset/legend and for multiple/composed inputs seem to be important and more frequent that we thought